### ISSUE TITLE
Fail to start language server

### ISSUE THREAD

**Issue by BenjaminHoegh** [psalm/psalm-vscode-plugin#275]
Type: <b>Bug</b>

# Behaviour

## Expected

The server to start?

## Actual

Fails/crash upon start

## Steps to reproduce:

1. No idea, I just start VSCode
2. No restart language server dosn't help

# Diagnostic data

-   PHP version: PHP 8.2.0 (cli) (built: Dec  8 2022 17:48:05) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.2.0, Copyright (c) Zend Technologies

-   Psalm version: 5.14.1

<details>

<summary>"Psalm Language Server" channel in the OUTPUT panel (Last 1000 lines)</summary>

<p>

```
[INFO  - 19.31.21] Starting language server
[INFO  - 19.31.22] Psalm Version was detected as 5.14.1
[ERROR  - 19.31.22] Problem parsing /Users/benjamin/Documents/Projekter/Privat/Open Source/ParsedownExtended/1.2/psalm.xml:
  Could not resolve config path to /Users/benjamin/Documents/Projekter/Privat/Open Source/ParsedownExtended/1.2//ParsedownExtended.php

Problem parsing /Users/benjamin/Documents/Projekter/Privat/Open Source/ParsedownExtended/1.2/psalm.xml:
  Could not resolve config path to /Users/benjamin/Documents/Projekter/Privat/Open Source/ParsedownExtended/1.2//ParsedownExtended.php

[Info  - 19.31.22] Connection to server got closed. Server will restart.
[INFO  - 19.31.22] Psalm Version was detected as 5.14.1
[ERROR  - 19.31.22] Problem parsing /Users/benjamin/Documents/Projekter/Privat/Open Source/ParsedownExtended/1.2/psalm.xml:
  Could not resolve config path to /Users/benjamin/Documents/Projekter/Privat/Open Source/ParsedownExtended/1.2//ParsedownExtended.php

Problem parsing /Users/benjamin/Documents/Projekter/Privat/Open Source/ParsedownExtended/1.2/psalm.xml:
  Could not resolve config path to /Users/benjamin/Documents/Projekter/Privat/Open Source/ParsedownExtended/1.2//ParsedownExtended.php

[Info  - 19.31.22] Connection to server got closed. Server will restart.
[INFO  - 19.31.22] Psalm Version was detected as 5.14.1
[ERROR  - 19.31.23] Problem parsing /Users/benjamin/Documents/Projekter/Privat/Open Source/ParsedownExtended/1.2/psalm.xml:
  Could not resolve config path to /Users/benjamin/Documents/Projekter/Privat/Open Source/ParsedownExtended/1.2//ParsedownExtended.php

Problem parsing /Users/benjamin/Documents/Projekter/Privat/Open Source/ParsedownExtended/1.2/psalm.xml:
  Could not resolve config path to /Users/benjamin/Documents/Projekter/Privat/Open Source/ParsedownExtended/1.2//ParsedownExtended.php

[Info  - 19.31.23] Connection to server got closed. Server will restart.
[INFO  - 19.31.23] Psalm Version was detected as 5.14.1
[ERROR  - 19.31.23] Problem parsing /Users/benjamin/Documents/Projekter/Privat/Open Source/ParsedownExtended/1.2/psalm.xml:
  Could not resolve config path to /Users/benjamin/Documents/Projekter/Privat/Open Source/ParsedownExtended/1.2//ParsedownExtended.php

Problem parsing /Users/benjamin/Documents/Projekter/Privat/Open Source/ParsedownExtended/1.2/psalm.xml:
  Could not resolve config path to /Users/benjamin/Documents/Projekter/Privat/Open Source/ParsedownExtended/1.2//ParsedownExtended.php

[Info  - 19.31.23] Connection to server got closed. Server will restart.
[INFO  - 19.31.23] Psalm Version was detected as 5.14.1
[ERROR  - 19.31.23] Problem parsing /Users/benjamin/Documents/Projekter/Privat/Open Source/ParsedownExtended/1.2/psalm.xml:
  Could not resolve config path to /Users/benjamin/Documents/Projekter/Privat/Open Source/ParsedownExtended/1.2//ParsedownExtended.php

Problem parsing /Users/benjamin/Documents/Projekter/Privat/Open Source/ParsedownExtended/1.2/psalm.xml:
  Could not resolve config path to /Users/benjamin/Documents/Projekter/Privat/Open Source/ParsedownExtended/1.2//ParsedownExtended.php

[Error - 19.31.23] Connection to server got closed. Server will not be restarted.
```

</p>
</details>

<details>

<summary>User Settings</summary>

<p>

```
phpExecutablePath: "/Applications/MAMP/bin/php/php8.2.0/bin/php"
phpExecutableArgs: ["-dxdebug.remote_autostart=0","-dxdebug.remote_enable=0","-dxdebug_profiler_enable=0"]
psalmVersion: null
psalmScriptPath: "vendor/vimeo/psalm/psalm-language-server"
psalmScriptArgs: []
disableAutoComplete: false
maxRestartCount: 5
unusedVariableDetection: false
enableVerbose: false
connectToServerWithTcp: false
enableUseIniDefaults: false
logLevel: "INFO"
analyzedFileExtensions: [{"scheme":"file","language":"php"},{"scheme":"untitled","language":"php"}]
configPaths: ["psalm.xml","psalm.xml.dist"]
hideStatusMessageWhenRunning: true
```

</p>
</details>

Extension version: 2.7.0
VS Code version: Code 1.81.1 (Universal) (6c3e3dba23e8fadc360aed75ce363ba185c49794, 2023-08-09T22:20:33.924Z)
OS version: Darwin arm64 23.0.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 Pro (10 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|9, 10, 9|
|Memory (System)|16.00GB (0.08GB free)|
|Process Argv|--crash-reporter-id 2d7dbd69-d9a5-49be-8683-60fb64bffb4a|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes516:30244333
vslsvsres303:30308271
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
pythonvs932:30410667
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
pynewext54:30695312
azure-dev_surveyone:30548225
vsccc:30803844
2e4cg342:30602488
89544117:30613380
2i9eh265:30646982
showlangstatbar:30737416
a2ce3375:30757347
pythonfmttext:30731395
pythoncmvfstrcf:30756944
fixshowwlkth:30771522
showindicator:30805244
pythongtdpath:30769146
i26e3531:30792625
gsofa:30804715
pythonnosmt12:30797651
pythonidxptcf:30805731
pythonnoceb:30805159
dsvsc013:30795093
dsvsc014:30804076
diffeditorv2:30812749

```

</details>

**Comment by BenjaminHoegh** [comment]
psalm.xml

<?xml version="1.0"?>
<psalm
    errorLevel="8"
    resolveFromConfigFile="true"
    useDocblockTypes="false"
>
    <projectFiles>
        <file name="ParsedownExtended.php" />
        <ignoreFiles>
            <directory name="vendor" />
        </ignoreFiles>
    </projectFiles>
</psalm>
